hooks/pre/30-upgrade.rb

```diff
@@ -102,6 +102,25 @@ def mark_qpid_cert_for_update
   end
 end
 
+# rubocop:disable MethodLength
+def recreate_event_queue
+  # piping output to /dev/null because on EL6 any qpid-stat/config commands gives a non fatal python traceback.
+  cert = '/etc/pki/katello/qpid_client_striped.crt'
+  `qpid-config --ssl-certificate #{cert} -b "amqps://localhost:5671" del exchange event --durable > /dev/null 2>&1`
```

Comment:
@daviddavis I could not use this with Kafo::Helpers.execute:

https://github.com/Katello/katello-installer/pull/487/files#diff-45212ce99fdfbd0351c83af485751584R119

Also its starting to wrap around which is not good because of how long.